site/content/contribute/developer-setup.md

```diff
@@ -19,8 +19,20 @@ If you're migrating from before the monorepo see the [migration notes]({{< ref "
 If you're developing plugins, see the plugin [developer setup]({{< ref "/integrate/plugins/developer-setup" >}}) documentation.
 {{</note>}}
 
+# Prerequisites for WSL Development Environment
+
+1. [Install WSL](https://learn.microsoft.com/en-us/windows/wsl/install) by running the following command as an administrator in PowerShell: `wsl --install`
+2. [Install Docker Desktop for Windows](https://learn.microsoft.com/en-us/windows/wsl/tutorials/wsl-containers#install-docker-desktop) on your Windows machine.
```

Comment:
I have had much better luck with installing Docker directly in the distribution after enabling systemd support: https://learn.microsoft.com/en-us/windows/wsl/wsl-config#systemd-support

Maybe we should suggest that instead or as well as docker desktop?

Reply:
From what I've experienced, devs on Windows generally have some kind of setup already on Windows which also includes the Docker Desktop. Also, the Docker Desktop plays nicely when you have multiple Linux setup as WSL. And it also _supports both_ Windows and WSL.
So, I would still like to keep the Docker Desktop as an option. But yeah, we can also add the "direct-docker" as an alternative.

Reply:
Sounds good.